Update create-external-data-source-transact-sql.md (#8188)
MicrosoftDocs/sql-docs: docs/t-sql/statements/create-external-data-source-transact-sql.md

PolyBase supports proxy based authentication for most external data sources. Create a database scoped credential to create the proxy account.

> Shared Access Signature (SAS) keys are supported for authenticating to Azure Data Lake Store Gen 2 Storage Accounts. Customers that want to authenticate using a Shared Access Signature must create a database scoped credential where IDENTITY = "Shared Access Signature" and enter a SAS token as the secret. Customers that want to authenticate using a Storage Account Key must use a database scoped credential where IDENTITY = “Storage Account Key” and use a storage key as the secret. 

Reviewer: ```suggestion
Shared Access Signature (SAS) keys are supported for authenticating to Azure Data Lake Store Gen 2 Storage Accounts. Customers that want to authenticate using a Shared Access Signature must create a database scoped credential where `IDENTITY = "Shared Access Signature"` and enter a SAS token as the secret. 
```

> If a customer creates a database scoped credential where IDENTITY = ‘Shared Access Signature’ and uses a storage key value as the secret, they will get the following error: 

Reviewer: ```suggestion
If you create a database scoped credential where `IDENTITY = "Shared Access Signature"` and uses a storage key value as the secret, you will get the following error message: 
```
